build/vsts-ci.yml

```diff
@@ -145,26 +157,25 @@ phases:
       msbuildVersion: 15.0
     continueOnError: false
 
-  - task: NuGetCommand@2
-    displayName: Publish Packages to VSTS Feed
+  - task: MSBuild@1
+    displayName: Publish Packages to MyGet Feed
     inputs:
-      command: push
-      packagesToPush: $(Build.SourcesDirectory)/bin/packages/**/*.nupkg;!$(Build.SourcesDirectory)/bin/packages/**/*.symbols.nupkg
-      nuGetFeedType: internal
-      feedPublish: MachineLearning
+      solution: build/publish.proj
+      msbuildArguments: /t:PublishPackages /p:NuGetFeedUrl=$(_NuGetFeedUrl) /p:NuGetApiKey=$(dotnet-myget-org-api-key)
```

Comment:
>dotnet-myget-org-api-key [](start = 93, length = 24)

Is it purposely not defined, or we have some kind of hidden storage somewhere?

Reply:
it will be defined in the build defination using variable groups.